web/src/main/java/com/softserve/osbb/controller/ApartmentController.java

```diff
@@ -142,64 +151,70 @@
         Apartment apartment = apartmentService.findOneApartmentByID(id);
         UserDTO user;
         Resource<UserDTO> userDTOResource;
-        if (apartment.getOwner() != null) {
-            user = UserDTOMapper.mapUserEntityToDTO(userService.findOne(apartment.getOwner()));
-            userDTOResource = addResourceLinkToUserDTO(user);
-        } else return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        
+        if (apartment.getOwner() == null) {
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
+
problem of multiple nulls in code :( 
to be solved during moving logic from controllers to services